<!DOCTYPE html>
<!-- https://latex.vercel.app/ -->
<html>
  <head>
    <link rel="stylesheet" href="https://latex.vercel.app/style.css">
    <title>Simone Ciccolella</title>

  </head>
  <body>
    <h1>Simone Ciccolella</h1>
    <p class="author">PostDoc at Bias Lab </br>May 2025</p>

    <div style="display:flex;justify-content:space-evenly;">
      <a href="./index.html"><b>Home</b></a>
      <a href="./publications.html">Publications</a>
      <a href="./data/simone_ciccolella_resume.pdf" target="_blank">Academic CV</a>
      <a href="https://scholar.google.com/citations?user=BLrs8WEAAAAJ&hl=en">Scholar</a>
      <a href="https://orcid.org/0000-0002-6469-4887">Orcid</a>
      <a href="https://github.com/sciccolella">Github</a>
    </div>

    <p>
    I am currently a PostDoc in the 
    <a href="https://algolab.eu/" target="_blank">BIAS Lab</a> 
    at the Computer Science Department -
    University of Study Milano-Bicocca, Italy.
    </p>

    <h3>Research Interest</h3>
    <p>
    My personal research activity has focused on the development of combinatorial
    and heuristic algorithms in Bioinformatics, with a strong focus 
    on praticality and efficiency. 
    I have paid special attention to implementation, validation and experimentation.
    </p>

    <p>
    In my reasearch I focused on <i>Combinatorial and Heuristic Algorithms</i> 
    for cancer and viral phylogenetics; 
    <i>Machine Learning</i> and <i>Deep Learning</i> for classification
    of images and tabular genomic data;
    <i>Graph-based algorithms, data structure and indexing</i> in the field
    of pangenomics.
    </p>

    <p>
    Lastly, I am interested in general algorithms and data structure,
    not necessarly related to bioinformatics,
    and their implementations, 
    especially regarding low-level coding optimizations in <code>C</code>.
    </p>

    <h3>Education</h3>
    <ul>
      <li>
        2022: Ph.D. in Computer Science, University of Study Milano-Bicocca.<br>
        Thesis:
        <a href="https://hdl.handle.net/10281/364980" target="_blank" >
          <em>Practical algorithms for Computational Phylogenetics</em>
        </a><br>
        Supervisor: 
        <a href="https://algolab.eu/people/gianluca-della-vedova/">Prof. Gianluca Della Vedova</a>
      </li>
      <li>
        2018: M.Sc. in Computer Science, University of Study Milano-Bicocca.<br>
        <!-- Thesis: <em>Inferring cancer progression from Single Cell Sequencing data while allowing loss of mutations</em><br> -->
        <!-- Supervisor:  -->
        <!-- <a href="https://algolab.eu/people/gianluca-della-vedova/">Prof. Gianluca Della Vedova</a>, University of Study Milano-Bicocca.<br> -->
        <!-- Co-supervisor:  -->
        <!-- <a href="http://physiology.med.cornell.edu/faculty/hajirasouliha/lab/">Prof. Iman Hajirasouliha</a>, Weill Cornell Medicine, New York. -->
      </li>
      <li>
        2016: B.Sc. in Computer Science, University of Study Milano-Bicocca.<br>
        <!-- Thesis: <em>A linear Programming Approach to reconstruct cancer phylogenies from multi-sample sequencing data</em><br> -->
        <!-- Supervisor: <a href="https://algolab.eu/people/gianluca-della-vedova/">Prof. Gianluca Della Vedova</a>, University of Study Milano-Bicocca.<br> -->
        <!-- Co-supervisor: <a href="https://algolab.eu/people/bonizzoni/">Prof. Paola Bonizzioni</a>, University of Study Milano-Bicocca. -->
      </li>
    </ul>

    <h3>International collaborations</h3>

    <ul>
      <li>
        <b>Georgia State University -- Atlanta, GA, USA</b>
        I have been collaborating with <i>Dr. Murray Patterson</i>
        on various projects regarding cancer phylogenomics,
        evolutionary studies of viruses, clustering and machine learning techniques for biological data, mostly regarding Single Cell sequencing data.
      </li>
      <li>
        <b>Weill Cornell Medicine -- New York City, NY, USA</b>
        I have been collaborating with <i>Prof. Iman Hajirasouilha</i> on the design and implementation of novel and
        efficient algorithms for cancer phylogeny reconstructions as well as novel evolutionary models
        to overcome the current limitations of the available approaches.
      </li>
      <li>
        <b>University of Tokyo -- Tokyo, Japan</b>
        I have been collaborating with <i>Prof. Kunihiko Sadakane</i> with goal of developing
        succinct data structures and efficient algorithms for strings and graphs.
        We worked on a graph indexing for pangenome graphs and are currently collaborating
        on block decomposition using string-based indices.
      </li>
    </ul>

  </body>
</html>
